test/__snapshots__/test.js.snap

```diff
@@ -195,14 +195,14 @@ module.exports = {
     return iface;
   }, // createInterface
 
-  mixedInto: [],
+  isMixedInto: [],
```

Comment:
Using `is` prefixes for non-boolean variables (an array in this case) is especially confusing.

Maybe this should have an underscore prefix since it appears to be an implementation detail?

So `_mixedIntoPredicates` perhaps?